dashboard/test/lib/services/curriculum_pdfs_test.rb

```diff
@@ -26,8 +25,9 @@ class CurriculumPdfsTest < ActiveSupport::TestCase
     end
 
     test 'get_pdfless_lessons excludes lessons without lesson plans' do
-      unit_with_lesson_plans = create :script, :with_lessons
-      unit_without_lesson_plans = create :script, :with_lessons
+      AWS::S3.stubs(:exists_in_bucket).returns(false)
+      unit_with_lesson_plans = create :script, :with_lessons, seeded_from: Time.now
+      unit_without_lesson_plans = create :script, :with_lessons, seeded_from: Time.now
```

Comment:
out of curiosity, is there a reason `seeded_from` needed to be added here?

Reply:
oh thanks for catching! There actually isn't; I added that in an effort to get this test passing again, but what I actually needed to do was adjust our stubbing strategy.